### PR TITLE
Delete leftover namespaces in down script

### PR DESCRIPTION
**Pull request by mtesseract**
## Description

When debugging issues in the e2e test suite it happens easily that we have to cleanup leftovers. In my opinion it makes sense to integrate this into `down.sh`.

## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Documentation added if necessary
- [ ] CI and all relevant tests are passing
- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.

## Test manual

TODO: Add manual testing efforts

```
# To run tests locally run:
make db/teardown db/setup db/migrate
make ocm/setup OCM_OFFLINE_TOKEN=<ocm-offline-token> OCM_ENV=development
make verify lint binary test test/integration
```

### CODE REVIEW

File: dev/env/scripts/down.sh [hunk 1]
```diff
@@ -14,3 +14,13 @@ port-forwarding stop db 5432 || true
 delete "${MANIFESTS_DIR}/db" || true
 delete "${MANIFESTS_DIR}/fleet-manager" || true
 delete "${MANIFESTS_DIR}/fleetshard-sync" || true
+
+central_namespaces=$($KUBECTL get namespace -o jsonpath='{range .items[?(@.status.phase == "Active")]}{.metadata.name}{"\n"}{end}' | grep '^rhacs-.*$')
```
Comment (SimonBaeumer) [inline dev/env/scripts/down.sh:18]: Where is `$KUBECTL` var declared?

**Review by SimonBaeumer — CHANGES_REQUESTED** [review]

File: dev/env/scripts/down.sh (hunk 1 above)
Comment (mtesseract) [reply dev/env/scripts/down.sh:18]: https://github.com/stackrox/acs-fleet-manager/blob/mc%2Fdebug-e2e-local/dev/env/scripts/lib.sh#L89

Comment (SimonBaeumer) [reply dev/env/scripts/down.sh:18]: Link is broken, but I see where it is coming from.
Should the `kubectl` in line 21 then be replaced?
https://github.com/stackrox/acs-fleet-manager/pull/340/files#diff-399711d07af14f32ea9544fecfd0cd1690b9505d0c21964c883a1b5f859fa3d8R21

**Review by SimonBaeumer — APPROVED** [review]

File: dev/env/scripts/down.sh (hunk 1 above)
Comment (SimonBaeumer) [reply dev/env/scripts/down.sh:18]: @mtesseract that was fast. 👍 Thank you!

Comment (mtesseract) [reply dev/env/scripts/down.sh:18]: Sorry, was on the wrong (non-pushed) branch... 🤦

**Comment by mtesseract** [comment]
@SimonBaeumer I managed to introduce a bug in in this tiny PR: `grep` would fail with a non-zero exit code in case no such namespaces are found and due to `set -e` being active the script would be aborted immediately. The `|| true` seems to be the common practice for enforcing a success exit code irregardless of matches being found or not.

**Comment by SimonBaeumer** [comment]
> @SimonBaeumer I managed to introduce a bug in in this tiny PR: `grep` would fail with a non-zero exit code in case no such namespaces are found and due to `set -e` being active the script would be aborted immediately. The `|| true` seems to be the common practice for enforcing a success exit code irregardless of matches being found or not.

Sounds good 👍

**Pull request merged**